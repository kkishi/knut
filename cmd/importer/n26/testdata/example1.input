"Date","Payee","Account number","Transaction type","Payment reference","Amount (EUR)","Amount (Foreign Currency)","Type Foreign Currency","Exchange Rate"
"2023-01-20","INTERACTIVE BROKERS LLC","XXXX","Income","2023-01-20","1000.0","","",""
"2023-01-21","GOOGLE*TEMPORARY HOLD","","MasterCard Payment","","-1.44","-200.0","JPY","0.0072"
