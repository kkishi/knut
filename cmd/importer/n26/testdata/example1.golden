2023-01-20 "INTERACTIVE BROKERS LLC"
Expenses:TBD           Liabilities:CreditCard       1000 EUR

2023-01-21 "GOOGLE*TEMPORARY HOLD"
Liabilities:CreditCard Expenses:TBD                 1.44 EUR

